digiroad2-geo/src/main/scala/fi/liikennevirasto/digiroad2/asset/asset.scala

```diff
@@ -670,7 +670,14 @@ case class PropertyValue(propertyValue: String, propertyDisplayValue: Option[Str
 case class DynamicPropertyValue(value: Any)
 case class ValidityPeriodValue(days: Int, startHour: Int, endHour: Int, startMinute: Int, endMinute: Int, periodType: Option[Int] = None)
 case class EnumeratedPropertyValue(propertyId: Long, publicId: String, propertyName: String, propertyType: String, required: Boolean = false, values: Seq[PointAssetValue]) extends AbstractProperty
-case class Position(lon: Double, lat: Double, linkId: Long, bearing: Option[Int])
+
+trait Coordinates {
+  def lon: Double
+  def lat: Double
+}
+case class PositionCoordinates(lon: Double, lat: Double) extends Coordinates
```

Comment:
What is the reason to create this?
This is practically the same as Point class.
Like we can see when it's used ->  "val point = Point(position.lon, position.lat)", in the end, we need to convert to Point to be used and is not used in other places.
Please start from the beginnings using class Point(lon, lat) and remove this case class, please.